src/main/java/seedu/duke/Duke.java

```diff
@@ -0,0 +1,77 @@
+package seedu.duke;
+
+import seedu.duke.command.Command;
+import seedu.duke.operations.Parser;
+import seedu.duke.operations.Storage;
+import seedu.duke.operations.TaskList;
+import seedu.duke.operations.Ui;
+import seedu.duke.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Duke is the main class for the chatbot application. It is designed to be
+ * able to take in inputs within command line, keep track of a list of tasks
+ * the user had typed.
+ *
+ * Duke is able to take the following commands:
+ * <ul>
+ *     <li>list</li>
+ *     <li>mark</li>
+ *     <li>unmark</li>
+ *     <li>delete</li>
+ *     <li>bye</li>
+ *     <li>todo</li>
+ *     <li>deadline</li>
+ *     <li>event</li>
+ * </ul>>
+ *
+ * @author Su Peigeng
+ */
+public class Duke {
+    private final TaskList tasks;
+    private final Storage storage;
+    private final Ui ui;
```

Comment:
Constants should be in all caps. 
```suggestion
    private final TaskList TASKS;
    private final Storage STORAGE;
    private final Ui UI;
```